utils/forest_shim/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "forest_shim"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+#fvm.workspace = true
+#fvm3 = { package = "fvm", version = "3.0.0-alpha.19" }
+fvm_shared.workspace = true
```

Comment:
Maybe also move `fvm_shared` out of workspace manifect, correct me if I'm wrong, with this shim crate,  I believe `fvm_shared` should not be referenced by other crates directly

Reply:
That's the plan. It'll have to be done incrementally as more as more types are moved out of `fvm_shared` and into `forest_shim`. Moving everything at once is a big mouthful.